app/steps/ui/deceased/dobknown/index.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const ValidationStep = require('../../../../core/steps/ValidationStep');
+const json = require('app/resources/en/translation/deceased/dobknown');
+const FormatName = require('app/utils/FormatName');
+
+class DeceasedDobKnown extends ValidationStep {
+
+    static getUrl() {
+        return '/deceased-dob-known';
+    }
+
+    nextStepOptions() {
+        const nextStepOptions = {
+            options: [
+                {key: 'dobknown', value: json.optionYes, choice: 'dobknown'}
+            ]
+        };
+        return nextStepOptions;
+    }
+
+    generateContent(ctx, formdata) {
+        const content = super.generateContent(ctx, formdata);
+        const deceasedName = FormatName.format(formdata.deceased);
+        content.question = content.question.replace('{deceasedName}', deceasedName);
+        return content;
+    }
```

Comment:
Replace this whole method with:
```
    getContextData(req) {
        const ctx = super.getContextData(req);
        const formdata = req.session.form;
        ctx.deceasedName = FormatName.format(formdata.deceased);
        return ctx;
    }
```

Reply:
And update unit test accordingly

Reply:
@shepda In conversation with David on Friday he confirmed that he would rather use the index.js file to replace the content variable that needs to be displayed with an appropriate unit test.